recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
-{% set version = "3.1" %}
-{% set sha256 = "979bf38db2c36193de49149aaea5c540d18e01ccc27cf76e2aff5606bd186722" %}
+{% set version = "3.2a" %}
```

Comment:
How is Conda going to handle this version number?

Reply:
According to https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html conda is following https://www.python.org/dev/peps/pep-0440/#version-scheme `[N!]N(.N)*[{a|b|rc}N][.postN][.devN]` ... that means 3.2a should be fine.

Reply:
Is this an alpha? If so, we should be following [CFEP 5]( https://github.com/conda-forge/cfep/blob/main/cfep-05.md )

Reply:
@dbast?

Reply:
tmux 3.2a is bugfix release of the 3.2 version (not an alpha version)... see https://github.com/tmux/tmux/releases